iterators.tex

```diff
@@ -6370,6 +6380,11 @@
 may want to specialize \tcode{is_sized_range} to override the default in the
 case of accidental conformance.\exitnote
 
+\ednote{\newtxt{This language needs to be stronger than ``free to/may want
+to specialize.'' If \tcode{size(foo const\&)} launches the missiles, then a
+user either \emph{must} specialize \tcode{is_sized_range<foo>} or we need
+specify to which library functions that user cannot pass a \tcode{foo}.}}
+
```

Comment:
Right. If a type does not meet the semantic requirements of a `SizedRange` (namely that `size(r)==distance(begin(r), end(r))`, modulo signedness), then `is_sized_range` must be specialized.